src/platform/graphics/constants.js

```diff
@@ -1021,12 +1121,29 @@ export const isIntegerPixelFormat = (format) => {
 };
 
 /**
+ * Returns the srgb equivalent format for the supplied linear format. If it does not exist, the input
+ * format is returned. For example for {@link PIXELFORMAT_RGBA8} the return value is
+ * {@link PIXELFORMAT_SRGBA8}.
+ *
+ * @param {number} format - The texture format.
+ * @returns {number} The format allowing linear sampling of the texture.
+ * @ignore
+ */
+export const pixelFormatLinearToGamma = (format) => {
+    return pixelFormatInfo.get(format)?.srgbFormat || format;
+};
+
+/**
+ * For a pixel format that stores color information, this function returns true if the texture
+ * sample is in sRGB space and needs to be decoded to linear space.
+ *
  * @param {number} format - The texture format.
- * @returns {boolean} - Whether sampling the texture with this format returns a linear value.
+ * @returns {boolean} Whether sampling the texture with this format returns a sRGB value.
  * @ignore
  */
-export const isLinearFormat = (format) => {
-    return !pixelFormatInfo.get(format)?.srgb;
+export const isColorSampleSrgb = (format) => {
```

Comment:
I would prefer something like `requiresManualGamma` or `requiresLegacyGamma` and then have docs refer to this feature as a legacy workaround for not having srgb pixel formats.

Reply:
`requiresManualGamma` it is, I like it.